contracts/src/interfaces/IGateway.sol

```diff
@@ -86,6 +86,9 @@ interface IGateway {
     /// @dev Check whether a token is registered
     function isTokenRegistered(address token) external view returns (bool);
 
+    /// @dev Get token id an ERC20 contract address.
```

Comment:
```suggestion
    /// @dev Get token id of an ERC20 contract address.
```

Reply:
Addressed in [38ea2b9](https://github.com/Snowfork/snowbridge/pull/1314/commits/38ea2b967413909cc827ac3a8f9c3396e2ccc277)